### ISSUE TITLE
bug: Screen rotation causes crash on Windows 10 Tablet (Ionic v1.2)

### ISSUE THREAD

**Issue by pjc2007** [driftyco/ionic#4768]
**Type**: <span ionic-type>bug</span>

**Platform**: <span ionic-platform>desktop</span>  <span ionic-webview>browser</span>

<span ionic-description>Hi, I have just used the submit issue, and used "desktop" for platform, but this is actually for Windows 10 tablet (Windows 10 not added here yet for version 1.2 that now supports?)

Anyway, I have just linked in the new version 1.2 into a Cordova project created in Visual studio. I have removed everything except for a single div in index.html, so I assume any blank app should be able to reproduce this.

Soon after a rotate of the screen,  (may a second or 2) the application just crashes. I had this in the previous version of the library (1.1), but still have it in version 1.2.

Content are at at http://codepen.io/peterjc/pen/VevMrJ

As you can see, there is nothing but a very basic blank app.

The only extra in index.html are the links.. (I link in winstore-jscompat.js as we still appear to need this for Windows 8)
`<head>
    <meta charset="utf-8" />
    <title>Tets app</title>

    <link href="lib/css/ionic.min.css" rel="stylesheet">
    <link href="css/index.css" rel="stylesheet" />

    <script src="lib/js/winstore-jscompat.js"></script>
    <script src="lib/js/ionic.bundle.min.js"></script>
   
    <script src="app/app.js"></script>
</head>`

Not sure what else to do try try and track down (but am willing to do anything suggested)
Thanks</span>

<span is-issue-template></span>

**Comment by Ionitron** [comment]
Greetings @pjc2007!

My sensors indicate that you need to update your issue through our [custom issue form](http://ionicframework.com/submit-issue/#/?iid=4768). We are now requiring all issues to be submitted this way, to ensure that we have all of the information necessary to fix them as quickly as possible.

**[Click Here To Update Your Issue](http://ionicframework.com/submit-issue/#/?iid=4768)**

I will have no choice but to close this issue if it is not resubmitted through the form. Please fill out the rest of the form, so that I may use my friendly robot powers to assist you.

Thank you!

<span ionitron-issue-resubmit></span>

**Comment by mlynch** [comment]
Hey @pjc2007, what windows tablet are you using, a Surface Pro? Would love to know exact specs so we can see about getting a device to test...

**Comment by pjc2007** [comment]
Hey Max, I've tried on two, one was a
HP Omni10, and the other was  a Del Venue 8 Pro.

Both have been upgraded to Windows 10. Since it has occurred on 2 different
devices it may occur on most?

Aside... If testing on a Windows device, the only other issue I have also
come across is the Select does not work... But I have posted to the forum
first as perhaps I didn't get the new mark up right.... Anyway one issue at
a time.

Let me know if there is anything I can do to help, run any test apps etc.

Thanks for the fast follow!

Pete

Hey @pjc2007 <https://github.com/pjc2007>, what windows tablet are you
using, a Surface Pro? Would love to know exact specs so we can see about
getting a device to test...

—
Reply to this email directly or view it on GitHub
<https://github.com/driftyco/ionic/issues/4768#issuecomment-164478933>.

**Comment by pjc2007** [comment]
Hi, I think I have got to the bottom of this, and it is not a bug!

Going back to a starter template, I had a couple of things missing.

To get it working, I made the following changes

1. I had the `<meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no, width=device-width">` missing, so I have added this back

2. In the config. I had the target set as Windows 8.1, so I changed this to Windows 10

3. I added back the security meta tag `<meta http-equiv="Content-Security-Policy" content="default-src 'self' data: gap: https://ssl.gstatic.com 'unsafe-eval'; style-src 'self' 'unsafe-inline'; media-src *">`

4. I removed the reference to `winstore-jscompat.js` (don't seem to need this now that I am targeting Windows 10).

I haven't tried it on the Del again yet, but now seems to work fine on the HP tablet. I think we can close this issue!
